src/app/containers/Metadata/index.test.jsx

```diff
@@ -28,29 +28,36 @@ const metadataProps = (
   metaTags,
   title,
   serviceConfig,
-) => ({
-  isAmp,
-  ampLink,
-  articleAuthor: serviceConfig.articleAuthor,
-  articleSection: null,
-  brandName: serviceConfig.brandName,
-  canonicalLink,
-  defaultImage: serviceConfig.defaultImage,
-  defaultImageAltText: serviceConfig.defaultImageAltText,
-  description,
-  facebookAdmin: 100004154058350,
-  facebookAppID: 1609039196070050,
-  lang,
-  locale: serviceConfig.locale,
-  metaTags,
-  themeColor: serviceConfig.themeColor,
-  timeFirstPublished: '2018-01-01T12:01:00.000Z',
-  timeLastPublished: '2018-01-01T13:00:00.000Z',
-  title,
-  twitterCreator: serviceConfig.twitterCreator,
-  twitterSite: serviceConfig.twitterSite,
-  type: 'article',
-});
+) => {
+  const appleTouchIcon = `http://foo.com/${
+    serviceConfig.service
+  }/images/icons/icon-192x192.png`;
+
+  return {
+    isAmp,
+    ampLink,
+    appleTouchIcon,
```

Comment:
Any reason why you moved this into a `return` statement? Could have achieved with:

```js
=> ({
  isAmp
  ampLink,
  articleAuthor: serviceConfig.articleAuthor,
  ...etc
  ...etc
  appleTouchIcon: `http://foo.com/${serviceConfig.service}/images/icons/icon-192x192.png`,
```

Reply:
`eslint` was not passing because this line was too long.

Reply:
Ok, but:

```suggestion
=> ({
  isAmp
  ampLink,
  articleAuthor: serviceConfig.articleAuthor,
  ...etc
  ...etc
  appleTouchIcon: `http://foo.com/${
    serviceConfig.service
  }/images/icons/icon-192x192.png`,
```

(without the `return`)

Don't mind 🤷‍♂️ but the creation of the `const` not strictly necessary.

Reply:
Happy to update, looks a lot cleaner

Reply:
Makes a smaller diff too 👍